ee/app/livechat-enterprise/server/api/priorities.js

```diff
@@ -0,0 +1,44 @@
+import { API } from '../../../../../app/api/server';
+import { findPriorities, findPriorityById, setPriorityToRoom } from './lib/priorities';
+
+API.v1.addRoute('livechat/priorities.list', { authRequired: true }, {
+	get() {
+		const { offset, count } = this.getPaginationItems();
+		const { sort } = this.parseJsonQuery();
+
+		return API.v1.success(Promise.await(findPriorities({
+			userId: this.userId,
+			pagination: {
+				offset,
+				count,
+				sort,
+			},
+		})));
+	},
+});
+
+API.v1.addRoute('livechat/priorities.getOne', { authRequired: true }, {
+	get() {
+		const { priorityId } = this.queryParams;
+
+		return API.v1.success(Promise.await(findPriorityById({
+			userId: this.userId,
+			priorityId,
+		})));
+	},
+});
+
+API.v1.addRoute('livechat/priorities.setPriorityToRoom', { authRequired: true }, {
```

Comment:
Let's create a new `room` method inside the [room](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/api/v1/room.js) file, ok?
We could create a PUT method called by `room.prioritize`, what do you think?